<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>siteblocker</class>
 <widget class="QMainWindow" name="siteblocker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>siteblocker</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>581</width>
      <height>371</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>271</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>270</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Add website to the list</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>301</width>
       <height>191</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Block selected websites</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
